pkg/config/template.go

```diff
@@ -1324,6 +1329,12 @@ const templateStringCrioImagePauseCommand = `# The command to run to have a cont
 
 `
 
+const templateStringCrioImagePinnedImages = `# List of container images to be skipped from the kubelet's garbage collection.
```

Comment:
non blocking: can we make it work with config reload via `systemctl reload crio`?